MapboxNavigation.podspec

```diff
@@ -44,7 +44,7 @@ Pod::Spec.new do |s|
   s.module_name = "MapboxNavigation"
 
   s.dependency "MapboxCoreNavigation", "#{s.version.to_s}"
-  s.dependency "Mapbox-iOS-SDK", "~> 5.5.0"
+  s.dependency "Mapbox-iOS-SDK", "= 5.5.0"
```

Comment:
CocoaPods wasn’t affected by the original issue, but I’m pinning the podspec too to ensure consistent dependencies regardless of the dependency manager used.